xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/store/XWikiHibernateStore.java

```diff
@@ -1246,13 +1245,12 @@ public void saveXWikiCollection(BaseCollection object, XWikiContext inputxcontex
                 // save object using the custom mapping
                 Map<String, Object> objmap = object.getCustomMappingMap();
                 handledProps = bclass.getCustomMappingPropertyList(context);
-                Session dynamicSession = session.getSession(EntityMode.MAP);
```

Comment:
This is not valid anymore and I'm not sure what will be the consequence of the removal.

Reply:
See https://hibernate.atlassian.net/browse/HHH-6330